ultrasonic/src/main/res/values/arrays.xml

```diff
@@ -236,5 +236,40 @@
         <item>@string/settings.playback.bluetooth_a2dp</item>
         <item>@string/settings.playback.bluetooth_disabled</item>
     </string-array>
+    <string-array name="languageNames" translatable="true">
+        <item>@string/language.default</item>
+        <item>@string/language.zh</item>
+        <item>@string/language.cs</item>
+        <item>@string/language.nl</item>
+        <item>@string/language.en</item>
+        <item>@string/language.fr</item>
+        <item>@string/language.de</item>
+        <item>@string/language.de_DE_1901</item>
+        <item>@string/language.hu</item>
+        <item>@string/language.it</item>
+        <item>@string/language.es</item>
+        <item>@string/language.pl</item>
+        <item>@string/language.pt</item>
+        <item>@string/language.pt_br</item>
+        <item>@string/language.ru</item>
+    </string-array>
 
+    <string-array name="languageValues" translatable="false">
+        <!--suppress CheckTagEmptyBody -->
+        <item></item>
+        <item>zh</item>
```

Comment:
We have two chinesse language Chinese (China) `zh-rCN` and Chinese (Taiwan) `zh-rTW`

Reply:
ok!